packages/dag4-network/src/dto/v2/snapshot.ts

```diff
@@ -2,12 +2,14 @@ import { Proof } from "./transaction";
 
 export type SnapshotV2 = {
   hash: string;
-  timestamp: string;
   ordinal: number;
   height: number;
   subHeight: number;
   lastSnapshotHash: string;
   blocks: string[];
+  epochProgress: number;
```

Comment:
This value is not present on all GL0 snapshots, please type as nullish => `value?: {type} | null`.